bindings_node/src/groups.rs

```diff
@@ -469,6 +469,37 @@ impl NapiGroup {
     Ok(group_name)
   }
 
+  #[napi]
+  pub async fn update_group_image_url_square(&self, group_image_url_square: String) -> Result<()> {
```

Comment:
```suggestion
  pub async fn update_group_image_url_square(&self, group_image_url_square: &str) -> Result<()> {
```

Reply:
If you can get away with it; it's typically better/more efficient to pass `str`-slices rather than allocated `String`s

Reply:
would love to understand this more. i was just matching the signature of `MlsGroup::update_group_image_url_square`.

Reply:
ahh. okay. you can probably get away with leaving that argument on your end to be passed by reference, a `&str` ; but you'll have to call `group_image_url_square.to_owned()` on the inner call (which defeats the benefit of not allocating) but you're reducing the places where we're passing `String` around if/when it is refactored to only allocate a `String` where its strictly necessary.   Most of the time, it's not.  And is better/more memory efficient to pass references.

understanding the differences between `&str` and `String`, really...